src/runners/express.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @fileoverview Runner for Express application
+ */
+const uuid4 = require('uuid4');
+const utils = require('../utils.js');
+const serverlessEvent = require('../proto/event_pb.js');
+const eventInterface = require('../event.js');
+const errorCode = require('../proto/error_code_pb.js');
+
+/**
+ * Creates an Event representing the running Express (runner)
+ * @param {Request} req The Express's request data
+ * @param {Int} startTime Runner start time
+ * @return {Object} The runner event
+ */
+function createRunner(req, startTime) {
+    const expressEvent = new serverlessEvent.Event([
+        `express-${uuid4()}`,
+        utils.createTimestampFromTime(startTime),
+        null,
+        'runner',
+        0,
+        errorCode.ErrorCode.OK,
+    ]);
+
+    const resource = new serverlessEvent.Resource([
+        req.path,
```

Comment:
Are you sure that it's a template and not an actual path? (which can get many different values).
What about connecting this event with the request? Maybe we need to add a strong ID to the response.
And the resource name in requests is the domain. If we wish to update it to the path, we need to do that in the backend.

Reply:
.